File: data/python/FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1_MAIN.py
if __name__ == '__main__':
    param = [
    (40,90,),
    (46,64,),
    (97,20,),
    (63,1,),
    (92,52,),
    (60,35,),
    (67,40,),
    (61,62,),
    (74,61,),
    (67,41,)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        try:
            if f_filled(*parameters_set) == f_gold(*parameters_set):
                print("FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1," + sys.argv[1] + "," + str(i) + ",SUCCESS")
            else:
                print("FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1," + sys.argv[1] + "," + str(i) + ",FAILURE")
        except:
            print("FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(40);
    param0.add(46);
    param0.add(97);
    param0.add(63);
    param0.add(92);
    param0.add(60);
    param0.add(67);
    param0.add(61);
    param0.add(74);
    param0.add(67);
    List<Integer> param1 = new ArrayList<>();
    param1.add(90);
    param1.add(64);
    param1.add(20);
    param1.add(1);
    param1.add(52);
    param1.add(35);
    param1.add(40);
    param1.add(62);
    param1.add(61);
    param1.add(41);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
	        if(f_filled(param0.get(i),param1.get(i)) == f_gold(param0.get(i),param1.get(i)))
	        {
                builder.append("FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1," + args[1] + "," + i + ",SUCCESS\n");
            } else {
                builder.append("FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1," + args[1] + "," + i + ",FAILURE\n");
            }
        } catch (Exception e) {
            builder.append("FIND_SUM_MODULO_K_FIRST_N_NATURAL_NUMBER_1," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
